store/src/java/com/zimbra/cs/gal/GalSyncToken.java

```diff
@@ -208,8 +208,17 @@ public boolean isEmpty() {
     public void merge(GalSyncToken that) {
         ZimbraLog.gal.debug("merging token "+this+" with "+that);
```

Comment:
I know this hasn't changes but worth changing to %s style here and elsewhere?  Especially as this is at debug level, so this usually creates unused objects.

Reply:
will change.